core/templates/services/questions-list.service.ts

```diff
@@ -84,16 +84,16 @@ export class QuestionsListService {
     if (
       (this._currentPage + 1) * num >
        this._questionSummariesForOneSkill.length &&
-       this._nextCursorForQuestions !== null && fetchMore) {
+       this._nextOffsetForQuestions !== null && fetchMore) {
```

Comment:
Why are we comparing it to `null`?

Reply:
Earlier the `cursor` was null when it was last page. So I updated next_offset to be null when the page was last page. So just replaced cursor with offset.